CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+## Setting up your working environment
+
+binary requires OCaml 4.05.0 or higher so you will need a corresponding opam switch
+which you can install by running:
+```
+$ opam switch create 4.05.0 ocaml-base-compiler.4.05.0
+```
+
+To clone the project's sources and install both its regular and test
+dependencies simply run:
+```
+$ git clone git@github.com:CraigFe/oskel.git
+$ cd oskel
+$ opam install -t --deps-only .
+```
+
+From there you can build all of the project's public libraries and executables
+with:
+```
+$ dune build @install
+```
+and run the test suite with:
+```
+$ dune runtest
+```
```

Comment:
This could be true of any project actually, if you use ppx_expect to unit test your library for instance.

Maybe it's worth exposing this extra information as an option so that people that don't use promotion as part of their workflow don't have to remove that section. What do you think?

Reply:
> This could be true of any project actually, if you use ppx_expect to unit test your library for instance.

True, but I think Oskel only needs to worry about this if it's actually generating the `ppx_expect` boilerplate (which I'm not against, it's just one of my workflows). 

I think it'd be nice if the project structures were always in sync with the help at the point of generation, which means either adding promotion information just for `kind=binary` or leaving it out completely; whichever you prefer.